samples/ClassroomLabs/Scripts/EthicalHacking/Setup-EthicalHacking.ps1

```diff
@@ -295,10 +296,11 @@ function Get-KaliLinuxDisk {
     }
 
     Write-Host "Extracting Kali Linux files from compressed file."
-    if ($null -eq (Get-ChildItem "$_kaliLinuxExtractedFilesFolder\Kali-Linux-2020.4-vmware-amd64.vmdk" -Recurse | Select-Object -expand FullName)) {
+    if ($null -eq (Get-ChildItem "$_kaliLinuxExtractedFilesFolder\$_kaliDownloadFileName" -Recurse | Select-Object -expand FullName)) {
```

Comment:
I think you wanted the "Kali-Linux-2021.4-vmware-amd64.7z"  not $_kaliDownloadedFileName.  The .7z file is there so it doesn't extract it.